ansible/dev.yml

```diff
@@ -0,0 +1,113 @@
+---
+- name: Set up Concord dependencies
+  hosts: all
+  tasks:
+    - name: Install apt tools
+      sudo: yes
+      apt: state=present name={{item}}
+      with_items:
+        - apt-transport-https
+        - software-properties-common
+
+    - name: Register apt repos
+      sudo: yes
+      apt_repository: repo='{{item}}' update_cache=yes state=present
+      with_items:
+        - ppa:ubuntu-toolchain-r/test
+        - ppa:webupd8team/java
+
+    - name: Auto accept oracle jdk license
+      sudo: yes
+      shell: echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | /usr/bin/debconf-set-selections
+
+    - name: Install apt packages
+      sudo: yes
+      apt: state=present name={{item}} update_cache=yes
+      with_items:
+        - git
+        - curl
+        - bzip2
+        - libntl0
+        - libmpfr4
+        - libssl1.0.0
+        - libgflags2
+        - libboost-thread1.55.0
+        - libboost-regex1.55.0
+        - libboost-program-options1.55.0
+        - libboost-system1.55.0
+        - libboost-filesystem1.55.0
+        - libboost-date-time1.55.0
+        - libboost-iostreams1.55.0
+        - libevent-2.0-5
+        - libunwind8
+        - libdouble-conversion1
+        - liblz4-1
+        - liblzma5
+        - libsnappy1
+        - libjemalloc1
+        - libgoogle-glog0
+        - zlib1g
+        - libbz2-1.0
+        - libarchive13
+        - libcurl3-nss
+        - libsvn1
+        - libsasl2-2
+        - libapr1
+        - libasan2
+        - lttng-tools
+        - liblttng-ust0
+        - libstdc++6=5.1.0-0ubuntu11~14.04.1
```

Comment:
Is this still true? 
can we add g++4.9 to this which will pkg up the libstdc++ -> need to add the tools/r repo. 

The reason for this is that without it our cpp does not compile.